test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -517,6 +517,7 @@ test
       })).eql(1);
   });
 
+// FIXME quarantined test OKTA-796308
 test
```

Comment:
Not a big deal but this test isn't skipped anymore

Reply:
saw the `.skip` annotation

```
...loopbackPollTimeoutMock).skip('new poll...
```

Reply:
👓 Gotta get my eyes checked

Reply:
Nope, placement was weird. Just verified locally that `.skip` (where it was) still causes test to be ignored